ac/src/intTest/README

```diff
@@ -0,0 +1,10 @@
+At this point, the integration tests can only be executed by CADC staff. Required resources in $RPS directory:
+    - ac-group-owner.pem : certificate file for owner of test group
+    - ac-group-member.pem : certificate file for member of test group
+    - ac-registered-user.pem : certificate file for registered user but not a member in any test group
+    - ac-anon-user.pem : certificate file for unregistered user
+    - ac-priv-user.pem : certificate file for privileged user capable of augmenting subject
+
+Other resources:
+    - $RPS/RsaSignaturePub.key file used to sign cookies on the service
```

Comment:
service needs the private key to sign (technically `ac` creates signed tokens and `access` repurposes them as cookies)

How do the tests pick one user in `.netrc` when there can be multiple entries? Just by looking for one that matches the target server name??

Reply:
hmmm, my .netrc would have my own (pdowler) entry, possibly even for my local deployments... maybe look at the VosiCapabilitiesTest in `cadc-test-vosi` (reg.git) because that also makes use of .netrc to test token auth (and that's used in many services)

Reply:
The account creation (and account requests creation) tests are currently skipped because they can't be undone. With a local disposable LDAP deployment we won't need the `.netrc` mechanism because the `ConfigUsers` could use the certificates to automatically create and configure accounts as need it and it would set the `passwordAuthUser` attribute required in the tests.